.github/issue_template.md

```diff
@@ -0,0 +1,11 @@
+## Expected Behavior
+<!--- If you're describing a bug, tell us what you expect to happen. -->
+
+## Current Behavior
+<!--- If you're describing a bug, what currently happens? -->
+
+## Steps to reproduce
+<!--- Provide a set of steps describing how to reproduce this bug. If you have a live example, provide the link below -->
+
+## Your Environment
+<!--- Provide details on your current environment you found the bug in -->
```

Comment:
Might be helpful to provide some pro-forma checkboxes here, too, so we make sure we capture Python versions, OS versions, and so on.